.eslintrc.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  parser: 'babel-eslint',
+  // parser: 'babel-eslint',
```

Comment:
why was this commented?

Reply:
It will be removed. There is already a parser on line 5 with typescript support. With this enabled it was throwing errors when things like 'as' are used.